### ISSUE TITLE
WCAG 2.0 2.1.1: Keyboard functionality: adding images

### ISSUE THREAD

**Issue by spudthebud** [ckeditor/ckeditor5#15165]
### Persona story 
This is fictitious to illustrate a problem. I work in a large organization creating content. Due to an injury, I have acquired hand-eye coordination impairment that means I cannot use the mouse to interact with computer content.  I rely on using the keyboard commands.

I would like to add an image to the CKEditor 5, then give it alt text.  The whole user task should be reviewed without reliance on a mouse.

The file browser link isn't keyboard accessible.

![image](https://github.com/ckeditor/ckeditor5/assets/8799391/f563f764-2975-4afb-ae5a-f2f937ef6d9d)

The add alt text control isn't accessible by keyboard

![image](https://github.com/ckeditor/ckeditor5/assets/8799391/72a9c141-bc6a-46c5-958c-373059c2bb33)

### ✔️ Expected result

Each step in the task of adding an image and giving it alt text should be possible using the keyboard only (WCAG 2.0 success criteria 2.1.1)

### ❌ Actual result

Can't complete the task

### ❓ Possible solution

_If you have ideas, you can list them here. Otherwise, you can delete this section._

## 📃 Other details

* Browser: Chrome
* OS: Windows 11
* First affected CKEditor version: 5
* Installed CKEditor plugins: ?

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by mgifford** [comment]
Definitely an accessibility challenge in the demo https://ckeditor.com/ckeditor-5/demo/feature-rich/

**Comment by Witoso** [comment]
Hi, thanks for the report. Unfortunately, those features are not native to CKE5. Both look like the ones provided by Drupal's core, and should be reported on their issue tracker.

**Comment by mgifford** [comment]
Why don't you post it to Drupal and share that link back here.

I can follow up with you there.
